src/modules/Hosts/Hosts/Views/MainPage.xaml

```diff
@@ -422,7 +387,26 @@
             Height="48"
             Grid.Row="1"
             IsActive="{x:Bind ViewModel.IsLoading, Mode=OneWay}" />
-        
+
+        <StackPanel
```

Comment:
There's something funky going on with the bottom spacing now. The left / right / bottom spacing between the card and window should be the same:

![image](https://github.com/microsoft/PowerToys/assets/9866362/09735a86-b0f7-4067-9dac-09a30ce4d424)

With error message:
![image](https://github.com/microsoft/PowerToys/assets/9866362/bf85124f-e5f4-4c34-af20-a2ca7be0afe7)

Reply:
Solved. They are both at the bottom of the list now. Only when they are both open, there's no space between them, but I have tried other ways and getting the margins right was difficult.